packages/statusbar-extension/src/index.ts

```diff
@@ -46,8 +50,15 @@ export const STATUSBAR_PLUGIN_ID = '@jupyterlab/statusbar-extension:plugin';
 const statusBar: JupyterFrontEndPlugin<IStatusBar> = {
   id: STATUSBAR_PLUGIN_ID,
   provides: IStatusBar,
+  requires: [ISettingRegistry, IMainMenu, ICommandPalette],
```

Comment:
Can you make the menu optional, and maybe even the command palette? Like in the `apputils-extension/src/index.ts` with the themesPaletteMenu extension? We're trying to make more things optional so plugins work in more contexts.